project/Build.scala

```diff
@@ -45,10 +45,35 @@ object Build extends AutoPlugin {
     shellout(bin +: args, cwd)
   }
 
-  implicit class FileOps(file: File) {
+  private implicit class FileOps(file: File) {
     def abs: String = file.getAbsolutePath
```

Comment:
(complete side-note: this is also available as [absolutePath](https://github.com/sbt/sbt/blob/v0.13.17/util/io/src/main/scala/sbt/Path.scala#L28) if you'd like)

Reply:
Very nice!! I'm going to use this right away. Thank you :)